extensions/wikia/TemplateDraft/TemplateDraftHooks.class.php

```diff
@@ -2,26 +2,54 @@
 
 class TemplateDraftHooks {
 
-	/*
-	 * property used in page_props table, construction:
-	 * * tc- - prefix for "template classifiaction"
-	 * * -marked- - signifies classification decision made by human
-	 * * -auto- - signifies classification decision made by AI
-	 * * -infobox - suffix denoting the type of template we identified
+	/**
+	 * Attaches a new module to right rail which is an entry point to convert a given template.
 	 *
-	 * @TODO move this someplace sensible, once available
+	 * @param array $railModuleList
+	 * @return bool
 	 */
-	const TEMPLATE_INFOBOX_PROP = 'tc-marked-infobox';
-
 	public static function onGetRailModuleList( Array &$railModuleList ) {
 		global $wgTitle;
 
 		if ( $wgTitle->getNamespace() === NS_TEMPLATE
-			&& Wikia::getProps( $wgTitle->getId(), self::TEMPLATE_INFOBOX_PROP ) !== 0
-		) {
-			$railModuleList[1502] = [ 'TemplateDraftModule', Index, null ];
+			&& Wikia::getProps( $wgTitle->getArticleID(), TemplateDraftController::TEMPLATE_INFOBOX_PROP ) !== 0 ) {
```

Comment:
Mi scusi! I have some issues with merging branches recently. I'll bring it back!